ssc/cmod_geothermal_costs.cpp

```diff
@@ -796,7 +797,7 @@ class cm_geothermal_costs : public compute_module
         double plant_om = non_labor_om_cost + labor_om_cost;
 
         //Field OM
-        double percent_drilling_cost_om = as_double("percent_drilling_cost_om");
+        double percent_drilling_cost_om = 1.5 / 100.0; //% always
```

Comment:
Is the drilling o and m finalized to a constant for all scenarios?

Reply:
Yes, for all relevant cases as confirmed by Erik.